### PR TITLE
Set up a react front end 

### PR DESCRIPTION
**Pull request by Hamayounali**
Implemented a connection between a Ruby on Rails back-end and React front-end. and displayed random greetings.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+function getAPIData() {
+  return axios.get(API_URL).then((response) => response.data)
+}
+
+function App() {
+  const [greetings, setGreetings] = useState([]);
+
+  useEffect(() => {
+    let mounted = true;
+    getAPIData().then((items) => {
```
Comment (youmari) [inline src/App.js]: - [ ] as required you should create a store and action to fetch from the API so please try to use redux in order to complete the requirements 
- Screenshot of the requirements 
Screenshot 2022-12-03 at 13 11 05

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @Hamayounali , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Great commit messages
✔️ PRO PR
✔️ No linter error 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rloterh — COMMENTED** [review]
Hi Hamayoun, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a frontend project (React), however, the link to the backend is missing!
- [ ] Kindly provide a link of the backend (Rails) in your frontend ReadMe, and similarly provide the link of the frontend(React) in the ReadMe of the backend.
- [ ] Your ReadMe says this project is a backend, however, it appears to be a frontend project. Kindly update your ReadMe accordingly.

Kindly address the above issues and then request another code review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by EricMbouwe — APPROVED** [review]
Hi @Hamayounali, 

Your project is complete! ⚡🔥  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🎉🔥
- Changes ✔
- Linters ✔

## Optional suggestions ☂

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**